packages/core/utils/src/sourcemap.js

```diff
@@ -2,7 +2,7 @@
 import type {FileSystem} from '@parcel/fs';
 import SourceMap from '@parcel/source-map';
 import path from 'path';
-import {normalizeSeparators} from './path';
+import {normalizeSeparators, isAbsolute} from './path';
```

Comment:
Not sure if we could use `path.isAbsolute` in this case because e.g. reading from `C:\` on macOS doesn't work anyway?